pkg/utils/utils.go

```diff
@@ -44,3 +45,24 @@ func Max(a, b uint64) uint64 {
 	}
 	return b
 }
+
+func ConvertHexaStringToIntWithBuffer(s string, buffer uint64) (uint64, error) {
```

Comment:
tên hàm này đọc hơi khó hiểu á, kiểu sẽ ko hiểu buffer dùng để làm gì. 

Hàm này ở trong ultis nên anh nghĩ là chỉ nên để phần parse hex rồi trả ra number thôi, phần tính toán finalized block number vẫn nên để lại trong hàm getFinalizedBlockNumber

Reply:
@tien-nguyen-sm Using buffer here for supporting delay some blocks when getting finalizedBlock from RPC node. Okay let me update the logic for easily reading.

Reply:
Updated function name